mkosi/log.py

```diff
@@ -6,8 +6,10 @@
 ARG_DEBUG: set[str] = set()
 
 
-def die(message: str, exception: Optional[Exception] = None) -> NoReturn:
+def die(message: str, exception: Optional[Exception] = None, hint: Optional[str] = None) -> NoReturn:
     MkosiPrinter.error(f"Error: {message}")
+    if hint:
+        MkosiPrinter.info(hint)
```

Comment:
Two of three users of hint put the message in parentheses, how about 
```suggestion
        MkosiPrinter.info(f"({hint})")
```
and adapt the message for secure boot keys?